public/app/plugins/datasource/loki/querybuilder/components/LokiQueryCodeEditor.tsx

```diff
@@ -39,6 +43,18 @@ export function LokiQueryCodeEditor({
         data={data}
         app={app}
         data-testid={testIds.editor}
+        onQueryType={async (query: string) => {
+          const timerange = datasource.getTimeRange();
+          const previousTimerange = undefined;
+          const previousQuery = undefined;
+
+          const update = shouldUpdateStats(query, previousQuery, timerange, previousTimerange);
+
+          if (update) {
+            const stats = await getStats(datasource, query);
+            setQueryStats(stats);
+          }
```

Comment:
Sorry, forgot to add this commend 😅

I think you should either also use `usePrevious` here or just directly call `getStats` as `update` will now always be `true`.